src/i18n/de.json

```diff
@@ -303,6 +303,15 @@
     "1.7.14.0.0": "Mehrere Filmgenres",
     "1.7.15.0.0": "Thriller"
   },
+  "steps": {
+    "offerTypes": {
```

Comment:
I meant in the actual step itself. Sorry for the confusion 😅

Reply:
No worries! I added it on the root level with the idea that we'll probably need it outside of the step as well. 
But adding it here also makes sense...

Reply:
I would opt for offerTypes on the root level since the translation won't change depending on the location where we use them

Reply:
I'm just a big fan of keeping all translations for 1 component under 1 key, yes there might be repetition, but this makes everything so much easier to move around etc

Reply:
Although I understand that this opens up questions like 'what do we do with the eventype labels?" etc

Reply:
It doesn't matter to me that much either way, so just merge it 😛

Reply:
I agree. Bigger fish to fry 😅

Reply:
🐟